packages/lesswrong/components/users/UsersEditForm.jsx

```diff
@@ -3,25 +3,40 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage, intlShape } from 'meteor/vulcan:i18n';
 import Users from 'meteor/vulcan:users';
-import FlatButton from 'material-ui/FlatButton';
+import Button from '@material-ui/core/Button';
 import { Accounts } from 'meteor/accounts-base';
 import { withRouter } from 'react-router'
+import Typography from '@material-ui/core/Typography';
+
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  header: {
+    margin: theme.spacing.unit * 2,
+    marginBottom: theme.spacing.unit * 4
+  },
+  resetButton: {
+    marginBottom:theme.spacing.unit * 4
+  }
+})
 
 const UsersEditForm = (props, context) => {
```

Comment:
I don't think we are using context here, so you should be able to just remove it.